### PR TITLE
Plat 161 update documentation

### PR DESCRIPTION
**Pull request by stefan-anmut**
Updates:
- modify scripts/init.sh to create a lock file if one doesn't exist
- README.md (typo fixes)
- folder for images, containing anmut logo
- {{cookiecutter.repo_name}}/README.rst
- {{cookiecutter.repo_name}}/CONTRIBUTING.rst
- {{cookiecutter.repo_name}}/docs/installation.rst
- {{cookiecutter.repo_name}}/docs/usage.rst changed to read in from USAGE.rst in root folder

Link to JIRA ticket:
https://anmutproductdev.atlassian.net/browse/PLAT-161?atlOrigin=eyJpIjoiMzdhOWIyODRhYjU5NGE5MGFiMTRiY2Q0ZjI1Njg2MjIiLCJwIjoiaiJ9

### CODE REVIEW

File: {{cookiecutter.repo_name}}/README.rst [hunk 1]
```diff
@@ -9,53 +13,50 @@
...
+Source Code
+===========
 
-.. seealso::
+The {{ cookiecutter.project_name }} source code can be found in the `Github repo`_ or in the `Distributions`_ folder within the Anmut Hub - Software Products SharePoint.
 
-	For more detailed installation instructions, see :ref:`installation`
+For more on installing from source, see :ref:`installation`.
 
-.. _SSH set up on GitHub: https://help.github.com/en/github/authenticating-to-github/connecting-to-github-with-ssh
```
Comment (aj-cloete) [inline {{cookiecutter.repo_name}}/README.rst:25]: I think it's important to leave this part in as it seems many people are not aware of how to do the SSH setup.  They don't even know about the option...

File: {{cookiecutter.repo_name}}/README.rst [hunk 2]
```diff
@@ -9,53 +13,50 @@
...
-This package was created with `Cookiecutter <https://cookiecutter.readthedocs.io>`_ and the `anmut-consulting/pipenv-cookiecutter <https://github.com/anmut-consulting/pipenv-cookiecutter>`_ project template.
+This package was created with `Cookiecutter`_ and the `anmut-consulting/pipenv-cookiecutter`_ project template.
+
+.. _Cookiecutter: https://cookiecutter.readthedocs.io
+.. _anmut-consulting/pipenv-cookiecutter: https://github.com/anmut-consulting/pipenv-cookiecutter
 
-.. |GitHub Test| image:: https://github.com/anmut-consulting/{{cookiecutter.repo_name}}/workflows/Test/badge.svg
-   :target: https://github.com/anmut-consulting/{{cookiecutter.repo_name}}/actions
+.. |GitHub Test| image:: https://github.com/anmut-consulting/vdva/workflows/Test/badge.svg
+   :target: https://github.com/anmut-consulting/vdva/actions
```
Comment (aj-cloete) [inline {{cookiecutter.repo_name}}/README.rst:59]: Whoops...these `vdva` parts should be replaced with `{{cookiecutter.repo_name}}` again.

File: {{cookiecutter.repo_name}}/docs/installation.rst [hunk 3]
```diff
@@ -6,34 +6,73 @@
...
+    $ source env/bin/activate
+
+And to deactivate the environment::
+
+    $ deactivate
+
+If you don't have pip installed, these `Python installation guides`_ can guide you through the process depending on your operating system and which python version you need.
+
+.. _pip: https://pip.pypa.io/en/stable/
+.. _SSH set up on GitHub: https://help.github.com/en/github/authenticating-to-github/connecting-to-github-with-ssh
```
Comment (aj-cloete) [inline {{cookiecutter.repo_name}}/docs/installation.rst:36]: Ah, I see you moved the SSH setup to here.  Okay ignore my comment about leaving it in the other file.

File: {{cookiecutter.repo_name}}/docs/installation.rst [hunk 4]
```diff
@@ -6,34 +6,73 @@
...
-The sources for {{ cookiecutter.project_name }} can be downloaded from the `Github repo`_.
+You can build any of the packages from source. Those involved in development may take this route to get developmental versions or alter source code.
+
+The sources for {{ cookiecutter.project_name }} can be downloaded from the:
+
+    * `Github repo`_, or
+    * `Distributions`_ SharePoint folder.
+
+.. _Github repo: https://github.com/anmut-consulting/{{ cookiecutter.repo_name }}
+.. _Distributions:
```
Comment (aj-cloete) [inline {{cookiecutter.repo_name}}/docs/installation.rst:72]: Let's populate the Distributions link to the link of the folder containing all the products and adapt the instructions to indicate that the user can search for the distribution in the relevant product folder (like vdva, data_condition, etc.)

File: {{cookiecutter.repo_name}}/docs/installation.rst [hunk 5]
```diff
@@ -45,14 +84,17 @@ Or download the `tarball`_:
 
 .. code-block:: console
 
-    $ curl  -OL https://github.com/anmut-consulting/{{ cookiecutter.repo_name }}/tarball/latest
+    $ curl -OL https://github.com/anmut-consulting/{{ cookiecutter.repo_name }}/tarball/latest
 
-Once you have a copy of the source, you can install it with:
+.. _tarball: https://github.com/anmut-consulting/{{ cookiecutter.repo_name }}/tarball/latest
```
Comment (aj-cloete) [inline {{cookiecutter.repo_name}}/docs/installation.rst:89]: Perhaps add a note stating that `/latest` can be replaced by the version they want (if that's the case...not too sure if github actually creates tarballs for every release).

File: {{cookiecutter.repo_name}}/docs/installation.rst [hunk 6]
```diff
@@ -45,14 +84,17 @@ Or download the `tarball`_:
...
 
-.. code-block:: console
+SharePoint
+++++++++++
 
-    $ python setup.py install
+Version zips and tarballs are also downloadable from the `Distributions`_ folder within the Anmut Hub - Software Products SharePoint.
```
Comment (aj-cloete) [inline {{cookiecutter.repo_name}}/docs/installation.rst:94]: See comment above about where to link.

**Review by aj-cloete — CHANGES_REQUESTED** [review]
Couple of quick changes requested but it looks good!

File: {{cookiecutter.repo_name}}/README.rst (hunk 2 above)
Comment (stefan-anmut) [reply {{cookiecutter.repo_name}}/README.rst:59]: great catch, thanks!

File: {{cookiecutter.repo_name}}/docs/installation.rst (hunk 5 above)
Comment (stefan-anmut) [reply {{cookiecutter.repo_name}}/docs/installation.rst:89]: yep, it works for any version release.

**Review by aj-cloete — APPROVED** [review]
Thanks @stefan-anmut ! Looks good to merge.

**Pull request merged**